src/Raven.Studio/wwwroot/App/views/database/tasks/editPeriodicBackupTask.html

```diff
@@ -40,6 +40,7 @@
                         <ul class="dropdown-menu" data-bind="foreach: backupOptions">
                             <li><a href="#" data-bind="text: $data, click: $parent.useBackupType.bind($parent, $data)"></a></li>
                         </ul>
+                        <div class="bg-info padding-left-xxs" data-bind="visible: $root.db.isSharded()"><small>Snapshot is not supported in sharded databases</small></div>
```

Comment:
in db level view we should use validation rules and prevent saving such configuration